CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+## Setup
+
+We recommend following setup:
+
+* make sure that [Composer](https://getcomposer.org) is installed
+* clone the [PHP_CodeSniffer repository](https://github.com/squizlabs/PHP_CodeSniffer)
+* run `composer install` inside the `PHP_CodeSniffer` directory to install required dependencies
+* symlink, copy or clone the [Symfony Coding Standard](https://github.com/djoos/Symfony-coding-standard) and symlink, copy or move its `Symfony` subdirectory to `PHP_CodeSniffer/src/Standards/Symfony` 
```

Comment:
I know this was copypasted from the README, but the first "symlink, copy or clone" could just get replaced by "get" now.